ioctl/cmd/action/action.go

```diff
@@ -214,3 +233,18 @@ func isBalanceEnough(address string, act action.SealedEnvelope) (err error) {
 	}
 	return
 }
+func signerIsExist(signer string) bool {
+	address, err := address.FromString(signer)
+	if err != nil {
+		return false
+	}
+	// find the account in keystore
+	ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+		keystore.StandardScryptN, keystore.StandardScryptP)
+	for _, account := range ks.Accounts() {
+		if bytes.Equal(address.Bytes(), account.Address.Bytes()) {
```

Comment:
Replace address with addr
```suggestion
		if address.Equal(addr, account.Address) {
```